ten-simple-rules-dockerfiles.Rmd

```diff
@@ -124,29 +124,29 @@ In this case, proceed with following the remaining rules to write your `Dockerfi
-It is even possible that images or `Dockerfile`s disappear.
+While some organizations can be trustworthy in terms of content, and reliably update containers with security fixes (e.g., `rocker/r-ver`), for most individual accounts that build containers, it's likely that containers will not be updated regularly.
+It is even possible that images or `Dockerfile`s may disappear.
 Therefore a good understanding of how base images and image tags work is crucial, as the tag that you choose has implications for your container build.
 You should know how to use `FROM` statements to trace all the way up to the original container base, an abstract base called `scratch`.
-Doing this kind of trace is essential to be able to be aware of all of the steps that were taken to generate your container and who took them.
+Doing this kind of trace is essential because it makes all of the steps that were taken to generate your container and who took them.
 If you want to build on a third party image, carefully consider the author/maintainer and _never_ use an image without a published `Dockerfile`.
 If you want to use an unofficial image, do save a copy of the Dockerfile(s) created by others.
-Alternatively copy relevant instructions into your own Dockerfile, acknowledge the source in a comment, and configure an automated build for your own image.
+Alternatively, copy relevant instructions into your own Dockerfile, acknowledge the source in a comment, and configure an automated build for your own image.
 Automated builds can be complex to set up, and details are out of scope of this article.
-The documentation and tools of container registries, e.g., [Docker Hub](https://docs.docker.com/docker-hub/builds/) or [GitLab](https://docs.gitlab.com/ee/user/packages/container_registry/index.html#build-and-push-images) instances, but also of CI platforms, e.g. [GitHub actions](https://github.com/actions/starter-workflows/tree/master/ci), a GitHub actions for `repo2docker` [@husain_repo2docker-action_2019; @scottyhq_repo2docker-githubci_2019], or [CircleCI](https://circleci.com/orbs/registry/orb/circleci/docker#commands-build), can help you get started.
+The documentation for, and toolsets provided by  container registries such as [Docker Hub](https://docs.docker.com/docker-hub/builds/) or [GitLab](https://docs.gitlab.com/ee/user/packages/container_registry/index.html#build-and-push-images) as well as CI platforms such as [GitHub actions](https://github.com/actions/starter-workflows/tree/master/ci) or [CircleCI](https://circleci.com/orbs/registry/orb/circleci/docker#commands-build) can help you get started; tools may also be available to automate builds using intermediate tools, such as the `repo2docker` Github action [@husain_repo2docker-action_2019; @scottyhq_repo2docker-githubci_2019],
 You should avoid publishing an image in a public registry with `docker push`, because it is opaquely breaking the linkage between `Dockerfile` and image.
```

Comment:
I think we need to do a check on this GitHub action - many (previously working actions) don't work after the API changes. If it still works, likely it needs to be called a GitHub workflow.

Reply:
okay if this is the linked repo - https://github.com/scottyhq/repo2docker-githubci it looks like it's been updated recently and should be good. Should we reference it as a GitHub workflow instead of action?